src/Sarif.Sdk.sln

```diff
@@ -122,11 +122,13 @@ Global
 		{66915F7B-9872-4390-AFED-DB8DE7B761BE}.Release|x86.ActiveCfg = Release|x86
 		{66915F7B-9872-4390-AFED-DB8DE7B761BE}.Release|x86.Build.0 = Release|x86
 		{F6727C72-C40C-4A6E-BAE0-23B4B5FE7DB1}.Debug|Any CPU.ActiveCfg = Debug|Win32
+		{F6727C72-C40C-4A6E-BAE0-23B4B5FE7DB1}.Debug|Any CPU.Build.0 = Debug|Win32
```

Comment:
This change ensures that the native project builds as part of Debug|Release configurations.